docs/Update-MarkdownHelpModule.md

```diff
@@ -65,51 +63,50 @@ Accept wildcard characters: False
 ```
 
 ### -LogAppend
-Don't overwrite log file, instead append to it.
-
+Indicates that this cmdlet appends information to the log instead overwriting it.
 
 ```yaml
 Type: SwitchParameter
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: False
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -LogPath
-Put log information into a provided file path.
-By default, VERBOSE stream is used for it.
+Specifies a file path for log information.
+By default, the cmdlet writes the VERBOSE stream to the log.
```

Comment:
vice verse, by default it uses VERBOSE stream as a log. I.e. if it's called with `-Verbose` user will see it on the screen. It also can be redirected as Verbose stream (`4>` I believe) and so on.